src/foam/mlang/mlang.js

```diff
@@ -3799,11 +3799,15 @@ foam.CLASS({
   constants: [
     {
       name: 'FALSE',
-      factory: function() { return foam.mlang.predicate.False.create() }
+      factory: function() { return foam.mlang.predicate.False.create(); }
     },
     {
       name: 'TRUE',
-      factory: function() { return foam.mlang.predicate.True.create() }
+      factory: function() { return foam.mlang.predicate.True.create(); }
+    },
+    {
+      name: 'EOD',
+      factory: function() { return foam.mlang.expr.EndOfDay.create().f(); }
```

Comment:
Should not call the .f(), otherwise you'll just record the  current timestamp and have the same problem we have now.